# HG changeset patch
# User Michael Goffioul <michael.goffioul@gmail.com>
# Date 1206397509 14400
#      Mon Mar 24 18:25:09 2008 -0400
# Branch release-3-0-x
# Node ID ed3315995d3ca3655cf7dfd0cccf719c3bd91fb3
# Parent  8408a69f6b059ad4bdbdd74738dd27de960c82de
pkg.m: adapt to changes in isspace for cell arrays of strings.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,9 +1,14 @@
-2008-02-19  John W. Eaton  <jwe@octave.org>
+2008-03-24  Michael Goffioul  <michael.goffioul@gmail.com>
+
+	* pkg/pkg.m (pkg:configure_make): Make it work with recent changes in
+	isspace handling with cell arrays of strings.
+
+2008-03-24  John W. Eaton  <jwe@octave.org>
 
 	* pkg/pkg.m: Style fixes.
 
 2008-03-24  Carlo de Falco  <carlo.defalco@gmail.com>
 
 	* pkg/pkg.m: Handle 'describe' command.
 	(parse_pkg_idx, print_package_description): New subfunctions.
 	
diff --git a/scripts/pkg/pkg.m b/scripts/pkg/pkg.m
--- a/scripts/pkg/pkg.m
+++ b/scripts/pkg/pkg.m
@@ -1291,33 +1291,33 @@ function configure_make (desc, packdir, 
       idx = [];
     else
       idx = cellfun (@(x) is_architecture_dependent (x), filenames);
     endif
     archdependent = filenames (idx);
     archindependent = filenames (!idx);
 
     ## Copy the files
-    if (! all (isspace (filenames)))
+    if (! all (isspace ([filenames{:}])))
 	if (! exist (instdir, "dir")) 
 	  mkdir (instdir);
 	endif
-	if (! all (isspace (archindependent)))
+	if (! all (isspace ([archindependent{:}])))
 	  if (verbose)
 	    printf ("copyfile");
 	    printf (" %s", archindependent{:});
 	    printf ("%s\n", instdir);
 	  endif
 	  [status, output] = copyfile (archindependent, instdir);
 	  if (status != 1)
 	    rm_rf (desc.dir);
 	    error ("Couldn't copy files from 'src' to 'inst': %s", output);
 	  endif
         endif
-	if (! all (isspace (archdependent)))
+	if (! all (isspace ([archdependent{:}])))
 	  if (verbose)
 	    printf ("copyfile");
 	    printf (" %s", archdependent{:});
 	    printf (" %s\n", archdir);
 	  endif
 	  if (! exist (archdir, "dir")) 
 	    mkdir (archdir);
 	  endif
